test/util/aggregator.go

```diff
@@ -162,7 +162,7 @@ func makeAPIService(gv schema.GroupVersion) *v1.APIService {
 	if !ok {
 		// if we aren't found, then we shouldn't register ourselves because it could result in a CRD group version
 		// being permanently stuck in the APIServices list.
-		klog.Infof("Skipping APIService creation for %v", gv)
+		klog.InfoS("Skipping APIService creation", "groupVersion", klog.KObj(gv))
```

Comment:
`gv` doesn't implement GetName() and GetNamespace(), so we cannot use klog.KObj here:

```suggestion
		klog.InfoS("Skipping APIService creation", "groupVersion", gv)
```

Reply:
Updated with latest push.